internal/nostr/models.go

```diff
@@ -14,26 +14,30 @@ const (
 	NIP_47_RESPONSE_KIND   = 23195
 
 	// state of request event
-	REQUEST_EVENT_PUBLISH_CONFIRMED   = "confirmed"
-	REQUEST_EVENT_PUBLISH_FAILED      = "failed"
+	REQUEST_EVENT_PUBLISH_CONFIRMED = "CONFIRMED"
+	REQUEST_EVENT_PUBLISH_FAILED    = "FAILED"
 )
 
 type Subscription struct {
 	ID            uint
-	RelayUrl      string       `validate:"required"`
+	RelayUrl      string            `validate:"required"`
 	WebhookUrl    string
 	Open          bool
-	Ids           *[]string     `gorm:"-"`
-	Kinds         *[]int        `gorm:"-"`
-	Authors       *[]string     `gorm:"-"` // WalletPubkey is included in this
-	Tags          *nostr.TagMap `gorm:"-"` // RequestEvent ID goes in the "e" tag
+	Ids           *[]string         `gorm:"-"`
+	Kinds         *[]int            `gorm:"-"`
+	Authors       *[]string         `gorm:"-"` // WalletPubkey is included in this
+	Tags          *nostr.TagMap     `gorm:"-"` // RequestEvent ID goes in the "e" tag
 	Since         time.Time
 	Until         time.Time
 	Limit         int
 	Search        string
 	CreatedAt     time.Time
 	UpdatedAt     time.Time
-	Uuid          string        `gorm:"type:uuid;default:gen_random_uuid()"`
+	Uuid          string            `gorm:"type:uuid;default:gen_random_uuid()"`
+	EventChan     chan *nostr.Event `gorm:"-"`
```

Comment:
I think it would be good to split the DB model from the application model. Then we do not need these gorm hints